.github/workflows/llm-connectivity.yml

```
@@ -1,79 +1,88 @@
-            echo '```json'
-            cat "$f"
-            echo '```'
-          } >>"$GITHUB_STEP_SUMMARY"
-          cat "$f"
-      - name: 计量账本验链 + 同步（metering-ledger 分支——cost-check LLM 预算通道数据源，ADR-0062）
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          bash pipeline/metering/metering-verify.sh --dir .metering
-          bash pipeline/metering/ledger-sync.sh --dir .metering --branch metering-ledger
+name: llm-connectivity
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)